packages/shared/components/molecules/AssetList.svelte

```diff
@@ -24,15 +35,14 @@
             <Text classes="text-left" type={TextType.h5}>{localize('general.assets')}</Text>
             <Filter filterStore={assetFilter} />
         </div>
-        <div class="flex-auto overflow-y-scroll h-1 -mr-5 pr-4 scroll-secondary">
-            <div class="-mr-4 overflow-x-visible space-y-2 ">
+        <div class="flex-auto h-1 -mr-5 pr-4">
+            <div class="-mr-4 h-full scroll-secondary">
                 {#if assets?.baseCoin || assets?.nativeTokens?.length > 0}
-                    {#if assets?.baseCoin}
-                        <AssetTile onClick={() => handleAssetTileClick(assets?.baseCoin)} asset={assets?.baseCoin} />
-                    {/if}
-                    {#each assets?.nativeTokens as asset}
-                        <AssetTile onClick={() => handleAssetTileClick(asset)} {asset} />
-                    {/each}
+                    <VirtualList items={assetList} let:item>
+                        <div class="mb-2">
+                            <AssetTile onClick={() => handleAssetTileClick(item)} asset={item} />
+                        </div>
```

Comment:
Can we pass classes into the list to apply the spacing rather than adding a asymmetrical margin to each item?